pkg/volume/plugins.go

```diff
@@ -169,6 +169,19 @@ func (spec *Spec) Name() string {
 // The binary should still use strong typing for this value when binding CLI values before they are passed as strings
 // in OtherAttributes.
 type VolumeConfig struct {
+	// RecyclerDefaultPod is the default pod used to scrub a persistent volume clean after its release.
+	// The default scrubber supplied by the system is a simple "rm -rf /* /.*" of a volume.
+	RecyclerDefaultPod *api.Pod
```

Comment:
why is it "recycler DEFAULT pod" and not "recycler pod" - what does "default" mean here?